pkgs/development/python-modules/python-trovo/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "python-trovo";
+  version = "0.1.4";
```

```suggestion
  version = "0.1.4";

  disabled = pythonOlder "3.8";
```

see https://codeberg.org/wolfangaukang/python-trovo/src/branch/main/pyproject.toml#L19

---

Done